engine/graphic/src/Texture.cpp

```diff
@@ -0,0 +1,86 @@
+#include "Texture.h"
+Texture::Texture(char* FileName)
+{
+	m_fileName = FileName;
+	imageId = 0;
```

Comment:
Why imageId equal to zero and there is no possibility to set it to 1,2,3,4? And what is difference between these options?